features/currencyexchange/domain/build.gradle.kts

```diff
@@ -22,7 +22,9 @@ kotlin {
                 implementation(libs.kotlinx.coroutines.core)
                 implementation(libs.kotlinx.datetime)
                 implementation(projects.common.coroutinesKtx)
-                implementation(projects.common.domain)           }
+                implementation(projects.common.domain)
+                implementation(projects.features.currencyexchange.data.remote.mindera.rest.schema)
+            }
```

Comment:
I might be wrong but the indentation seems weird to me 😅 seems like the curly bracket is not aligned with the dependencies

Reply:
It's aliged. Please check the image

<img width="362" alt="Screenshot 2024-06-07 at 4 48 40 PM" src="https://github.com/Mindera/Android-KMP-Template/assets/162277514/bd814651-dcfe-438f-a138-310da9360005">